lib/active_merchant/billing/gateways/creditcall.rb

```diff
@@ -26,10 +26,16 @@ def purchase(money, payment_method, options={})
           r.process { capture(money, r.authorization, options) }
         end
 
+        if multi_response.responses[1].nil?
+          merged_params = multi_response.primary_response.params
+        else
+          merged_params =  multi_response.responses[0].params.merge(multi_response.responses[1].params)
```

Comment:
This would overwrite any `auth` params of the same name from the `capture`. Just want to make sure that's safe in this context?